README.md

```diff
@@ -1,6 +1,6 @@
 [![AsyncAPI Banner and Logo](./public/img/logos/github-repobanner-website.png)](https://www.asyncapi.com)
 
----
+
```

Comment:
Removing the new line to remove the extra space between logo and socials. Seems redundant. Other than that. LGTM
```suggestion
```

Reply:
😆 Thanks for the review man